dj_rest_auth/views.py

```diff
@@ -70,14 +69,12 @@ def get_response_serializer(self):
 
     def login(self):
         self.user = self.serializer.validated_data['user']
+        token_model = get_token_model()
```

Comment:
To enable overriding settings in the tests, we must get the token model at runtime instead of at import time. The `get_token_model` enables this. This is similar to how you can set a custom user model in Django.